test/bench/kafka-sink-avro-debezium/README.md

```diff
@@ -0,0 +1,30 @@
+# Kafka Avro Debezium Sink Benchmark
+
+This benchmark tests our ability to write messages out to Kafka. In it's
+current configuration, it uses a topic with only one partition as input and a
+topic with only one partition as output. We use only one input topic because
+that is enough to saturate writing. We use only one output topic because the
+current Kafka sink will only ever write from on timely worker.
```

Comment:
nit:
```suggestion
This benchmark tests our ability to write messages out to Kafka. In its
current configuration, it uses a topic with only one partition as input and a
topic with only one partition as output. We use only one input topic because
that is enough to saturate writing. We use only one output topic because the
current Kafka sink will only ever write from one timely worker.
```